.github/workflows/r-ci.yml

```diff
@@ -79,13 +79,13 @@ jobs:
       #  if: ${{ matrix.os != 'macOS-latest' }}
       #  run: cd apis/r && Rscript -e "options(bspm.version.check=TRUE); install.packages('tiledb', repos = c('https://eddelbuettel.r-universe.dev/bin/linux/jammy/4.3/', 'https://cloud.r-project.org'))"
 
-      - name: Install r-universe build of SeuratObject (macOS)
-        if: ${{ matrix.os == 'macOS-latest' }}
-        run: cd apis/r && Rscript -e "install.packages('SeuratObject', repos = c('https://mojaveazure.r-universe.dev', 'https://cloud.r-project.org'))"
+      #- name: Install r-universe build of SeuratObject (macOS)
```

Comment:
@mojaveazure do these two for SeuratObject need to be "in" or "out"? The (very helpful) comment material above tells us under which conditions we need the two tiledb-r stanzas in or out ... but what about these two SeuratObject stanzas?

Reply:
These two install development versions of SeuratObject to ensure compatibility with up-coming. However, they are not necessary as the current version (SeuratObject v5.0.1) has all the fixes that these stanzars were originally written for

Reply:
@eddelbuettel in that case let's please remove those stanzas.